### PR TITLE
[Keyboard Layouts] Add missing characters in German and Greek keyboards

### PR DESCRIPTION
**Pull request by hudokkow**
Requested at http://forum.kodi.tv/showthread.php?tid=216980

### CODE REVIEW

**Comment by da-anda** [comment]
thanks. I just realized that § is missing in German and many latin keyboard layouts as well (just as on iOS and Android keyboards - so better don't use this character in passwords you have to enter in a smartphone ;) )

**Comment by hudokkow** [comment]
Updated.
@da-anda, do you know other latin keyboards that use the "§" so I can add it and make a new PR?
I'm using a Portuguese keyboard and that character doesn't even exist.

**Comment by Mettbrot** [comment]
Hey! Just saw that the € is missing too. (maybe next to $)
Don't you think it would be better to show the ß on the 'normal' page right next to 0 for example?

**Comment by da-anda** [comment]
@hudokkow according to wikipedia (https://en.wikipedia.org/wiki/Keyboard_layout) "§" is also used on greek keyboard layout.

@Mettbrot good catch with the €

**Comment by hudokkow** [comment]
@da-anda updated.

**Comment by tamland** [comment]
€ is a symbol so why shouldn't that be on the symbol page? Perhaps it's time to agree on a standard here because I see symbols missing on several layout. So either follow the actual keyboard layout or create one with *all* symbols.

**Comment by MartijnKaijser** [comment]
€ doesn't belong on the letter part imo and should be at the symbol one.

**Comment by hudokkow** [comment]
OK. Lets agree on a standard and go from there.
I'll redo the commits as soon as we have a few more opinions.

**Comment by Montellese** [comment]
I'd follow the actual keyboard layouts as much as possible because that's what users are familiar with. Any "special" characters that need a modifier key on a normal keyboard should go into symbols.
I'm pretty sure that looking at the android/ios keyboard layout/approach won't hurt either.

**Comment by Mettbrot** [comment]
@hudokkow I only meant the ß on the letter page. € right next to $ is fine :)
Also µ next to m seems strange. I don't know why its there on some keyboards. It's not like we use greek letters that much in german..

**Comment by da-anda** [comment]
@hudokkow mind rebasing this as it can't be merged atm. Thanks.

**Comment by hudokkow** [comment]
@da-anda, a bit late but done.

File: system/keyboardlayouts/german.xml [hunk 1]
```diff
@@ -6,19 +6,19 @@ Default font lacks support for all characters
 <keyboardlayouts>
   <layout language="German" layout="QWERTZ">
     <keyboard>
-      <row>1234567890</row>
+      <row>1234567890ß</row>
       <row>qwertzuiopü</row>
       <row>asdfghjklöä</row>
-      <row>yxcvbnm</row>
+      <row>yxcvbnmμ</row>
```
Comment (da-anda) [inline system/keyboardlayouts/german.xml]: "μ" should not be there IMO, rather move it to the symbols (I didn't even know it's on my keyboard ;) )

**Comment by hudokkow** [comment]
Done.

**Pull request merged**